File: frontend/web_ccadmin_ecommerce/src/app/enterprise/sale/model/dto/CheckoutDeliveryDto.ts
export class CheckoutDeliveryDto {
  public DeliveryTypeCod: string = '';
  public ClientAddressID: number | null = null;
  public IsThirdParty: string = 'N';
  public Names: string = '';
  public DocumentType: string = '01';
  public DocumentNumber: string = '';
  public Phone: string = '';
  public Email: string = '';
  public Address: string = '';
  public Reference: string = '';
  public UbigeoCod: string = '';
  public Latitude: number | null = null;
  public Longitude: number | null = null;
  public EstimatedDistanceKm: number | null = null;
  public Instructions: string = '';
  public ScheduledFrom: string = '';
  public ScheduledTo: string = '';
}

File: backend/ws_wa_store_ccadmin/src/main/java/com/ccadmin/app/delivery/model/dto/CheckoutDeliveryDto.java
package com.ccadmin.app.delivery.model.dto;

import java.math.BigDecimal;

public class CheckoutDeliveryDto {

    public String DeliveryTypeCod;
    public String IsThirdParty = "N";
    public String Names;
    public String DocumentType;
    public String DocumentNumber;
    public String Phone;
    public String Email;
    public String Address;
    public String Reference;
    public String UbigeoCod;
    public BigDecimal Latitude;
    public BigDecimal Longitude;
    public BigDecimal EstimatedDistanceKm;
    public String Instructions;
    public String ScheduledFrom;
    public String ScheduledTo;
}
